socorro/external/postgresql/report.py

```diff
@@ -84,7 +84,7 @@ def get_list(self, **kwargs):
                     '%s is not a recognized sort order key' % sort_order['key']
                 )
             sort_order['direction'] = 'ASC'
-            if 'reverse' in kwargs:
+            if str(kwargs.get('reverse', '')).lower() == 'true':
```

Comment:
Because before, if you do `reverse=false` in the URL this would mean that `'reverse'` would not exist in `kwargs`. Now it does, but it's `False`. What a mess, eh?